worker/src/core/loaders/message-worker/util/govsg.class.ts

```diff
@@ -0,0 +1,132 @@
+import { loggerWithLabel } from '@core/logger'
+import { map } from 'lodash'
+import { QueryTypes, Transaction } from 'sequelize'
+import { Sequelize } from 'sequelize-typescript'
+import OnpremWhatsappClient from '@shared/clients/onprem-whatsapp-client.class'
+import { PhoneNumberService } from '@core/services/phone-number.service'
+import config from '@core/config'
+
+const logger = loggerWithLabel(module)
+
+class Govsg {
+  private workerId: string
+  private connection: Sequelize
+  private client: OnpremWhatsappClient
+
+  constructor(workerId: string, connection: Sequelize) {
+    this.workerId = workerId
+    this.connection = connection
+    this.client = new OnpremWhatsappClient()
+  }
+
+  async enqueueMessages(jobId: number, campaignId: number): Promise<void> {
+    await this.connection.transaction(async (transaction: Transaction) => {
+      await this.connection.query('SELECT enqueue_messages_govsg(:jobId);', {
+        replacements: { jobId },
+        type: QueryTypes.SELECT,
+        transaction,
+      })
+
+      await this.connection.query('SELECT update_stats_govsg(:campaignId);', {
+        replacements: { campaignId },
+        type: QueryTypes.SELECT,
+        transaction,
+      })
+    })
+    logger.info({
+      message: 'Enqueued govsg messages',
+      workerId: this.workerId,
+      jobId,
+      action: 'enqueueMessages',
+    })
+  }
+
+  async getMessages(
+    jobId: number,
```

Comment:
**[discussion with Stanley]**
- we will do batch query to flamingo db, put it as a field in the message object to specify which API client to use
- sendMessage function callsite includes this information 

As such:
- use query builder because Sequelize not available on workers
- two different queries, batch queries for efficiency on workers for campaign sending, single query for transactional sending